### ISSUE TITLE
index.js:271 Uncaught (in promise) TypeError: Cannot read property 'children' of null     at NewsTicker._callee5$

### ISSUE THREAD

**Issue by velociraptor111** [ahmetcanaydemir/react-advanced-news-ticker#1]
Thanks for creating this awesome library! I came across an issue though. As soon as I changed the URL to render a different Page component i came across this error:

```
Uncaught (in promise) TypeError: Cannot read property 'children' of null
    at NewsTicker._callee5$ (index.js:271)
    at tryCatch (runtime.js:63)
    at Generator.invoke [as _invoke] (runtime.js:293)
    at Generator.next (runtime.js:118)
    at step (index.js:22)
    at index.js:22
    at new Promise (<anonymous>)
    at NewsTicker.<anonymous> (index.js:22)
    at NewsTicker._handleUpAnimation (index.js:317)
    at NewsTicker._callee6$ (index.js:350)
    at tryCatch (runtime.js:63)
    at Generator.invoke [as _invoke] (runtime.js:293)
    at Generator.next (runtime.js:118)
    at step (index.js:22)
    at index.js:22
```

Seems to me like somehow the `.next` function is still being called although I am rendering a completely different component. What do you think? Thank you!

**Comment by ahmetcanaydemir** [comment]
@velociraptor111 I am glad you like it. Also, you catched a good bug which I didn't realize. Thank you for your feedback! I released the 1.0.1 version, it should fix this bug. If you see any bug again, please don't hesitate to open an issue :)

**Comment by velociraptor111** [comment]
That's great ! All fixed now.
